vimish-fold.el

```diff
@@ -205,7 +218,10 @@ If PREFIX is not NIL, setup fringe for every line."
                  (propertize "…" 'display
                              (list vimish-fold-indication-mode
                                    'empty-line
-                                   'vimish-fold-fringe)))))
+                                   (if (vimish-fold--nested-vimish-overlay-p
```

Comment:
This does not work for me. Nested folds use the same fringe as non-nested folds: `vimish-fold-fringe`.